features/step_definitions/tag_set_steps.rb

```diff
@@ -150,6 +150,13 @@
   visit tag_set_path(tagset)
 end
 
+When /^I view a tag set that does not exist/ do
+  id = 1
+  tagset = OwnedTagSet.find_by_id(id)
+  tagset&.destroy
```

Comment:
Hound might have given you some bad advice here -- I _think_ `&.` only works in Ruby 2.3.0 and up, and we're still on 2.2.5, but if it passes on Travis, that's the important thing!